README.rst

```diff
@@ -100,9 +103,12 @@ blueprint directly in the normal way, e.g.:
         app.run()
 
 
+Note that you need to call the ``rq_dashboard`` blueprint AFTER the config, as explained here_ with that your config will overwrite the default config in ``rq_dashboard``
```

Comment:
I'm not sure what you mean by 
> you need to call the rq_dashboard blueprint

Maybe It's worth elaborating on this a bit more?

Reply:
Sure, let me try in another way

In your README, you have an example a blueprint that calls the configuration from your module:

```
app = Flask(__name__)
app.config.from_object(rq_dashboard.default_settings)
app.register_blueprint(rq_dashboard.blueprint, url_prefix="/rq")
```

I was naive, and stay with the `app.config.from_object(rq_dashboard.default_settings)` even when I have my other config file, so my app.py was like:

```
app.config.from_object(object_name)
app.register_blueprint(rq_dashboard.blueprint, url_prefix="/rq") 
app.config.from_object(rq_dashboard.default_settings)
```
with that, the load from your config reset my config and it didn't work

I just figured out now that this was the real problem :man_facepalming:  I don't need your config line just my

with that, you can close this PR :+1:  or if you think that is a way to explain my mistake to help others

Just to make things clear, my solution - and this PR - was because when I changed the order it worked:

```
app.config.from_object(rq_dashboard.default_settings)
app.config.from_object(object_name)
app.register_blueprint(rq_dashboard.blueprint, url_prefix="/rq") 
```

I was thinking that loading your configuration was essential to `rq-dashboard` work properly.

Thanks for your time and `rq-dashboard`

Reply:
I don't know why, but If i dont put the `app.config.from_object(rq_dashboard.default_settings)` the `rq-dashboard` didn't work, dont show any queues on dashboard.

Reply:
Ok, I got It. I believe this will be obsolete with upcoming release, since I just have removed dependency on default_settings and removed everything from there (except for DEBUG=False)

Reply:
@lorn does It work for you without importing default settings now?

Reply:
No
without the settings the dashboard is like this:

```
    #app.config.from_object(rq_dashboard.default_settings)
    app.config.from_object(object_name)
```
![withoutsettings](https://i.imgur.com/fe8ybpC.png)

with the settings:

```
    app.config.from_object(rq_dashboard.default_settings)
    app.config.from_object(object_name)
```

![withsettings](https://i.imgur.com/HbHO0lL.png)

Reply:
Yep, that was because the version was not released at the time.
Starting from 0.5.3 everything should work flawlessly for this scenario.

Reply:
Awesome, everything work with this last version :+1:

Reply:
I'm not sure about the flow, if you think that everything is ok you can close this pull request.
Thanks

Reply:
You can always close everything yourself, but since I'm already here... :)
Thank you for pointing this out. We are currently cleaning up dashboard behavior and It was important addition to uncluttering the code.